dev-tools/node.d.ts

```diff
@@ -0,0 +1,6 @@
+declare namespace NodeJS {
+    interface Global {
+		Dex: any
+		toId: any
```

Comment:
You can't write out the function signature here? It's just `(in: any) => string`

Reply:
Yeah you can. It's still needed in the other file though to be able to use <code>toId</code> without <code>global.</code>.